scripts/release.sh

```diff
@@ -423,9 +423,17 @@ build_helm_deps() {
   fi
 
   echo "Updating helm chart dependencies ..."
-  # Helm chart directory path -- /scripts --> /chart
-  chart_dir="$SCRIPT_DIR/../chart"
-  dep_chart_dir="$chart_dir/charts"
+
+  # Helm chart directory path -- /scripts --> /chart (or /charts)
+  local chart_dir=
```

Comment:
Let's instead specify this, like we do for IMAGES:
```
IMAGES="agents.core agents.ha.node agents.ha.cluster operators.diskpool rest csi.controller csi.node"
```
and do like
```
CHART=.....
```

Reply:
Done. PTAL.